examples/program_generator/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+#![no_std]
+
+use codec::alloc::vec::Vec;
+use gstd::{prog::ProgramGenerator, CodeHash};
+
+fn salt_uniqueness_test() {
+    let n = 10;
+    let salts: Vec<Vec<u8>> = (0..n).map(|_| ProgramGenerator::get_salt()).collect();
```

Comment:
We may let the compiler infer the type automatically.
```suggestion
    let salts: Vec<_> = (0..n).map(|_| ProgramGenerator::get_salt()).collect();
```

Reply:
done